public/locales/es/translation.json

```diff
@@ -42,6 +42,19 @@
     "siteMap": "Site Map",
     "copyright": "Copyright © 2020 State of California"
   },
+  "links": {
+    "edd-bpo": "https://www.edd.ca.gov/Benefit_Programs_Online_Espanol.htm",
+    "edd-di-claim": "https://edd.ca.gov/Disability/How_to_File_a_DI_Claim_in_SDI_Online.htm",
```

Comment:
this matches the doc and there doesn't appear to be a spanish version of this page